ajax/libs/two.js/package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "two.js",
   "filename": "two.min.js",
-  "version": "0.6.0",
+  "version": "0.7.0-beta.1",
   "description": "A two-dimensional drawing api meant for modern browsers.",
-  "homepage": "http://jonobr1.github.com/two.js",
+  "homepage": "https://two.js.org/",
```

Comment:
As homepage update is not directly related with the version update, this should be separated into another commit